docs/concepts/09_glossary.md

```diff
@@ -0,0 +1,157 @@
+
+A request made by a third party to an attester to verify the authenticity of a claim made by a claimer. The request for attestation includes the specific claim type and attributes to be verified, and it can be used to generate verifiable credentials that can be shared with others.
+
+### Trust Anchors
+
+Entities that are trusted to issue or verify claims on the KILT network, such as governments, educational institutions, or professional organizations.
+
+### Trust Market
+
+A market that operates on trust and reputation in addition to financial incentives, where buyers and sellers exchange goods or services based on established reputation through digital platforms.
+While trust markets offer benefits such as reducing the need for intermediaries, they also face challenges that need to be addressed to maintain trust and fairness in transactions.
+
+### Transport-Agnostic Messaging Layer
+
+A messaging system that is not dependent on any particular communication protocol or technology.
+It allows different systems or applications to communicate with each other regardless of the underlying transport protocol used, providing a standardized way of exchanging messages across different platforms and technologies.
+
+### Verifier
+
+A person, organization, or system that checks the validity and authenticity of an individual's credentials or qualifications.
+Verifiers play a critical role in building trust and ensuring that credentials are accurate and reliable.
+
+## Ecosystem Terms
+
+### Blockchain Technology
+
+A type of distributed ledger technology that allows multiple parties to have a synchronized, transparent, and immutable record of transactions.
+It uses cryptographic techniques to secure and verify transactions, and it does not require a central authority or intermediaries.
+
+### Decentralized Network
+
+A network of computers or nodes that operate without a central authority or control.
+In a decentralized network, each node has equal control over the network, and decisions are made through a consensus mechanism, rather than by a single entity or group.
+
+### Distributed Ledger Technology (DLT)
+
+A type of digital database that stores information across a network of computers or nodes.
+It allows multiple parties to have a synchronized, transparent, and immutable record of transactions, without the need for a central authority or intermediaries.
+
+### Extrinsics
+
+A transaction that originates from an external account and affects the state of the blockchain.
+It can be used to execute actions on the network, such as transferring funds, making governance decisions, use functionality of the parachain or interacting with smart contracts. More details about Extrinsics can be found at the [offical Polkadot documentation](https://wiki.polkadot.network/docs/learn-extrinsics)
+
+### Parachain
+
+A specialized blockchain that runs on top of the Polkadot network and is connected to the main network through a relay chain.
```

Comment:
What is the main network? Isn't that the relaychain?

Reply:
```suggestion
A specialized blockchain that runs on top of the Polkadot network and is connected through the relay chain.
```